src/analysis/connectivity/score_inputs.sql

```diff
@@ -0,0 +1,2636 @@
+----------------------------------------
+-- INPUTS
+-- location: neighborhood
+----------------------------------------
+DROP TABLE IF EXISTS generated.neighborhood_score_inputs;
+
+CREATE TABLE generated.neighborhood_score_inputs (
+    id SERIAL PRIMARY KEY,
+    category TEXT,
+    score_name TEXT,
+    score NUMERIC(16,4),
+    notes TEXT,
+    human_explanation TEXT,
+    use_pop BOOLEAN,
+    use_emp BOOLEAN,
+    use_k12 BOOLEAN,
+    use_tech BOOLEAN,
+    use_univ BOOLEAN,
+    use_doctor BOOLEAN,
+    use_dentist BOOLEAN,
+    use_hospital BOOLEAN,
+    use_pharmacy BOOLEAN,
+    use_retail BOOLEAN,
+    use_grocery BOOLEAN,
+    use_social_svcs BOOLEAN,
+    use_parks BOOLEAN,
+    use_trails BOOLEAN,
+    use_comm_ctrs BOOLEAN,
+    use_transit BOOLEAN
+);
+
+
+-------------------------------------
+-- population
+-------------------------------------
+-- median pop access ratio
+INSERT INTO generated.neighborhood_score_inputs (
+    category, score_name, score, notes, human_explanation, use_pop
+)
+SELECT  'People',
+        'Median ratio of access to population',
+        quantile(pop_ratio,0.5),
+        regexp_replace('Ratio of population accessible by low stress
```

Comment:
This raises the question of where the website is going to be pulling descriptions of each score from. Would it make the most sense to include that as a column in this table (as we were doing previously?) Or is that best handled separate from the tabular data?

In other words, should I be adding a "human readable" column that contains text ready to published on the website?